libraries/standard/http/cbmc/proofs/HTTPClient_Send/HTTPClient_Send_harness.c

```diff
@@ -35,8 +35,8 @@ void harness()
 
     /* Ideally, this should be allocated in the heap but doing so makes CBMC
```

Comment:
Does the comment no longer apply?

Reply:
No it does not i will add deleting this as part of the SIM wrote for the HTTP proof quality.